datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java

```diff
@@ -440,11 +441,14 @@ private void validateRazEnablement(SdxClusterRequest sdxClusterRequest, Detailed
             if (!razEntitlementEnabled) {
                 validationBuilder.error("Provisioning Ranger Raz is not enabled for this account.");
             }
-            if (!AZURE.name().equalsIgnoreCase(environment.getCloudPlatform())) {
-                validationBuilder.error("Provisioning Ranger Raz is only valid for Microsoft Azure.");
+            if (!(AWS.name().equalsIgnoreCase(environment.getCloudPlatform()) || AZURE.name().equalsIgnoreCase(environment.getCloudPlatform()))) {
+                validationBuilder.error("Provisioning Ranger Raz is only valid for Amazon Web Services and Microsoft Azure.");
             }
-            if (!isRazSupported(sdxClusterRequest.getRuntime())) {
-                validationBuilder.error("Provisioning Ranger Raz is only valid for CM version >= 7.2.1 and not " + sdxClusterRequest.getRuntime());
+            if (!isRazSupported(sdxClusterRequest.getRuntime(), environment.getCloudPlatform())) {
+                boolean aws = AWS.name().equalsIgnoreCase(environment.getCloudPlatform());
+                String errorMsg = aws ? "Provisioning Ranger Raz on Amazon Web Services is only valid for CM version >= 7.2.2 and not " :
```

Comment:
we could do AWS.equals(platform) ? "" : "";  where platform is defined as my previous comment.

Reply:
Thanks, addressed.